homebrew_challenge/src/components/HeatMap/Heatmap.jsx

```diff
@@ -208,6 +210,29 @@ function Heatmap({
     }
   }
 
+  function dateRangeTableCell() {
+
+    function formatDate(date) {
+      return format(date, "dd MMM yyyy");
+
+    }
+
+    const dates = dataset["dates"];
+    let startDate = Date.parse(dates[0]);
+    let endDate = Date.parse(dates[dates.length - 1]);
+
+    if (VALUETYPE_DEATHS === valueType) {
+      endDate = addDays(endDate, 6);
```

Comment:
Might be worth commenting why you're adding 6 days here

Reply:
comment added